app/backend/articles/models.py

```diff
@@ -0,0 +1,12 @@
+from backend.models import CustomUser
+from django.db import models
+
+# Create your models here.
+
+class Article(models.Model):
+    title = models.CharField(max_length=100, null=False)
+    author = models.ForeignKey(CustomUser, null=False, on_delete=models.CASCADE)
+    body = models.TextField(null=False)
```

Comment:
- Using TextField to handle variable and long lengths of text may cause a problem. 
- It poses a danger such that someone writes millions of lines of text as a body, e.g. a 50000 pages of article. It may disrupt the database flow and storage. 
- Maybe we should add a max_limit to the body parts of Articles and Posts/Comments.

Reply:
* Noted that 'max_length' will be added